packages/wonder-blocks-labeled-field/src/components/flexible-labeled-field.tsx

```diff
@@ -0,0 +1,67 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import {semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {BodyText} from "@khanacademy/wonder-blocks-typography";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+type Props = {
+    id?: string;
+    testId?: string;
+    label: React.ReactNode;
+    field: React.ReactElement;
+    helperTextAbove?: React.ReactNode;
+    helperTextBelow?: React.ReactNode;
+};
+export const FlexibleLabeledField = (props: Props) => {
+    const {label, field, helperTextAbove, helperTextBelow} = props;
+
+    const fieldId = React.useId();
+    const helperTextAboveId = React.useId();
+    const helperTextBelowId = React.useId();
+    const hasError = field.props.error;
+    const isDisabled = field.props.disabled;
+
+    function renderField() {
+        return React.cloneElement(field, {
+            id: fieldId,
+            "aria-describedby": [helperTextAboveId, helperTextBelowId]
+                .filter(Boolean)
+                .join(" "),
+        });
```

Comment:
In LabeledField, we would also set the `error` and `readOnly` props on the `field` here. Now, since we don't know if there is an `errorMessage` or `readOnlyMessage`, we leave it up to the consumer to set those props on the field properly. 

I like this because it narrows down what the LabeledField component does